### PR TITLE
docs(appium): refer to Python client repository instead

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

Current example in appium.io for python client is old. This PR just refers to the client readme instead to up-to-date the example. This will reduce https://github.com/appium/python-client/issues/1077 this kind of issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [Contributing Guide](https://appium.io/docs/en/latest/contributing/)
- [x] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/appium/docs/en/quickstart/test-java.md [hunk 1]
```diff
@@ -17,7 +17,6 @@ you could simply use the [AppiumDriver](https://github.com/appium/java-client/bl
 or build your custom derivatives from it. Check the [Drivers Support](https://github.com/appium/java-client#drivers-support)
 article to learn more about the current driver class implementations.
 
-Follow the [Usage Examples](https://github.com/appium/java-client#usage-examples) article in order understand
-how to invoke Java client features from your test framework.
+Follow the [Usage Examples](https://github.com/appium/java-client#usage-examples) article in order understand to how to invoke Java client features from your test framework.
```
Comment (mykola-mokhnach) [inline packages/appium/docs/en/quickstart/test-java.md:20]: ..article to understand how to...

File: packages/appium/docs/en/quickstart/test-py.md [hunk 2]
```diff
@@ -14,47 +14,9 @@ so installing the Appium Python Client includes the selenium binding.
 pip install Appium-Python-Client
 ```
 
-This example uses Python's built-in `unittest` module, though you can use any Python test framework you want.
-The Appium Python client adds the `appium:` vendor prefix automatically.
-You usually do not need to worry about the prefix.
+Then, simply you could use the client features by importing webdriver module as
+`from appium import webdriver` in your test code.
+Follow the [Usage](https://github.com/appium/python-client#usage) article in order to
+understand how to use the Python client features further.
```
Comment (mykola-mokhnach) [inline packages/appium/docs/en/quickstart/test-py.md:20]: ...article to understand how to use further Python client features.

**Comment by eglitise** [comment]
I'm not really in favor of this approach. Having some sample code for each language client in the main Appium documentation provides an excellent starting point for new users, which was also mentioned [in the PR that added the dotnet sample code](https://github.com/appium/appium/pull/19371). This approach also offers an easy way of comparing the syntax across the clients for the same functionality.
Additionally, the example provided in the Python client repo readme, while useful for its scalable approach, is notably more complex, and not fully applicable unless the user wants to use the same codebase for iOS and Android tests.
I would strongly recommend just updating the example code instead.

**Comment by KazuCocoa** [comment]
Oh, i noticed the original ticket https://github.com/appium/python-client/issues/1077 referred to Appium 2.0 version, which was the old Python client version. The latest one has proper one https://appium.io/docs/en/2.12/quickstart/test-py/ so this is `appium.io`'s referencing client version thing. Closing this ticket as we already have latest code in the latest version of appium.io

**Pull request closed without merging**